src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -144,13 +144,30 @@ for further details.
 	--sidecar-position-top: calc(
 		var(--sticky-bars-height) + var(--main-area-padding-top)
 	);
+	--sidecar-top-slot-variant-disclosure-height: 0;
 	--sidecar-viewport-bottom-spacing: 24px;
 	--max-sidecar-height: calc(
-		100vh - var(--sidecar-position-top) - var(--sidecar-viewport-bottom-spacing)
+		100vh - var(--sidecar-position-top) - var(--sidecar-viewport-bottom-spacing) -
+			var(--sidecar-top-slot-variant-disclosure-height)
 	);
 
 	display: none;
 
+	/** 
+	* NOTE! 
+	* Hard-coded height value to account for the height of the variants
+	* dropdown disclosure. We should find a more permanent workaround
+	* for the topslot to work for every generic component. 
+	* 
+	* This solves an issue where the variant dropdown disclosure shouldn't
+	* be in the scrollable sidecarslot area, and need its dropdown content
+	* to render outside of that scrollable area. This workaround retains the 
+	* scroll / scrim style behavior of the current sidecar slot.
+	*/
+	&.topSlotDropdown {
```

Comment:
Short term workaround, should refactor to a longer term solution when we need to use this slot for anything but the variant disclosure. https://app.asana.com/0/1204333057896641/1204669563792095

Reply:
😞 Not my favorite fix but running out of time here. Open to leaving this open for @zchsh to take a look at next week before we fully ship to find a more holistic workaround.

Reply:
This seems like a great solution for now, and dig the follow-up task! 🙌 I'll def take a look this week, thanks for flagging 🙇‍♂️